Examples/arcs-make

```diff
@@ -119,6 +143,7 @@ help:
 	@echo ""	
 	@echo "	ARCS Specific Options:"
 	@echo "	s		minimum sequence identity [98]"
+	@echo " cut		cut length for long reads (for arcs-long only) [250]"
```

Comment:
I guess this is relevant for ARCS and ARKS now that we have both modes? So should probably move to "Common options"